variables.tf

```diff
@@ -55,18 +55,26 @@ variable "location" {
   description = "The default location used for resources in this Landing Zone."
 }
 
+variable "terraform_version" {
+  type        = string
+  description = "Terraform version to install."
+  default     = "1.9.2"

```

Comment:
If we don't put a default value we can force people to always lookup the latest version, because it will fail without a version. Otherwise people might forget and will keep deploying 1.9.2 even when it's outdated

Reply:
Yes, good idea. Changed.